controllers/articles.controllers.js

```diff
@@ -1,14 +1,29 @@
-const { selectArticleById } = require("../models/articles.models");
+const {
+  selectArticleById,
+  selectAllArticles,
+  selectCommentCount,
+} = require("../models/articles.models");
 
 function getArticleById(req, res, next) {
   const articleId = req.params.article_id;
   selectArticleById(articleId)
-    .then((articleData) => {
-      res.status(200).send({ articleData });
+    .then((article) => {
+      res.status(200).send({ article });
     })
     .catch((err) => {
       next(err);
     });
 }
 
-module.exports = { getArticleById };
+function getAllArticles(req, res, next) {
+  const { sort_by, order } = req.query;
+  selectAllArticles(sort_by, order)
+    .then((allArticles) => {
+      res.status(200).send(allArticles);
```

Comment:
If you are send article when selecting one (with the id) then articles might be a better key to send articles. No biggie though 👍